README.md

```diff
@@ -0,0 +1,54 @@
+# uu2nano
+
+Simple tool for converting well-known [UUID](https://datatracker.ietf.org/doc/html/rfc4122)
+format (36 chars) to [nanoid](https://pypi.org/project/nanoid/) format (21 chars).
+
+
+## Details of conversion
+
+UUID is basically a 128-bit number. Some of those bits contain metadata including
+the version, on which the content of the remaining bits depends.
+For example, for UUIDv4 all 122 unspecified bits should contain random entropy.
+But more interesting that regardless of version, 2 bits in each UUID are fixed.
+That is why any UUID of any version could be encoded as 126 bits.
+
+UUIDs are encoded in hex format (4 bit per symbol) and 4 hyphens are added
+as separators, which leads to 128/4 + 4 = 36 characters in the string representation.
+
+For example: `492b6acb-05c7-4914-b139-253070a085e9`
+
+Nanoid is encoded using 64 URL-safe symbols (`A-Za-z0-9_-`) which makes possible
+to store 6 bits of information per symbol. To store 126 UUID bits, only 
+126 / 6 = 21 characters are used in the string representation.
+
+For example: `ggHEMKl5gfh2T7h-KC6lD`
+
+
+## Usage
+
+Convert one to another:
+
+```python
+import uuid
+from uu2nano import fix_uuid, nanoid_to_uuid, uuid_to_nanoid
+
+uu = uuid.uuid4()
+nano = uuid_to_nanoid(uu)
+assert uu == nanoid_to_uuid(nano)
+```
+
+If you receive UUIDs from an untrusted place, there is a chance that eventually
```

Comment:
Would probably not call such place `untrusted`, it's more about how it treats UUIDs.

If you receive UUIDs from a source, that generates them as simple random numbers, there is a chance that ....